### PR TITLE
2638: Add stylelint for native

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Add stylelint for native for linting of our styles (with styled-components).

### Proposed changes

- Add stylelint for native
- Move dependencies to root package
- Execute stylelint in CI
- Fix a few occurrences
- Tidy up

### Side effects

None.

### Resolved issues

Fixes: #2638.

---

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]
Looks good, not tested

File: native/src/routes/TuNewsDetail.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import LoadingErrorHandler from './LoadingErrorHandler'
 const HeaderImageWrapper = styled.View`
   margin: 20px 16px 0;
   border-radius: 8px;
-  background-color: rgba(2, 121, 166, 0.4);
+  background-color: rgb(2 121 166 / 40%);
```
Comment (ztefanie) [inline native/src/routes/TuNewsDetail.tsx:20]:
```suggestion
  background-color: rgb(2 121 166 / 0.4);
```

Comment (ztefanie) [reply native/src/routes/TuNewsDetail.tsx:20]: Will not wokr with the percentage for me.

**Review by ztefanie — APPROVED** [review]
Tested on Android emulator. Please add a comment to the issue / PR, so Martin know he needs to check for style changes / bugs.

File: native/src/routes/TuNewsDetail.tsx (hunk 1 above)
Comment (steffenkleinle) [reply native/src/routes/TuNewsDetail.tsx]: The whole notation doesn't work for me, fixed. Thanks for the catch!

**Pull request merged**